### ISSUE TITLE
PRM tool docs don't actually describe how to create the .tar.gz

### ISSUE THREAD

**Issue by binford2k** [puppetlabs/prm#121]
The docs describe how to structure the directory of a tool, and they say that you can install from a .tar.gz archive. But I couldn't figure out exactly how to create that archive in the expected format from the directory I created. Every time I tried installing it, I got

```
$ ~/.puppetlabs/prm/prm install addlicense-1.0.0.tar.gz
Error: Could not extract TAR from GZIP (addlicense-1.0.0.tar.gz): open /var/folders/km/cz24f0_j1bs4sh93q82sftp00000gq/T/468434073: is a directory
```

**Comment by petergmurphy** [comment]
Hi Ben,

I have tested creating the `tar.gz` archive locally and found that two commands must be executed to make it.

```bash
tar -cvf example.tar path/to/directory
gzip example.tar
```

Let me know if that works for you. If it does, I'll append it to the documentation.

**Comment by michaeltlombardi** [comment]
Fixed in #123, will be deployed shortly.
